<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q::vtk::InteractorStyleWindowLevel</class>
 <widget class="QWidget" name="Q::vtk::InteractorStyleWindowLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InteractorStyleWindowLevel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Window:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="horizontalSliderWindow">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxWindow"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Level:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSlider" name="horizontalSliderLevel">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxLevel"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderWindow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxWindow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxWindow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderWindow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderLevel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxLevel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxLevel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderLevel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
